conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -4,16 +4,10 @@
 
 environment:
 
-  CONDA_INSTALL_LOCN: "C:\\conda"
-
   # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
   # /E:ON and /V:ON options are not enabled in the batch script intepreter
   # See: http://stackoverflow.com/a/13751649/163740
-  CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
-
-  # We set a default Python version for the miniconda that is to be installed. This can be
-  # overridden in the matrix definition where appropriate.
-  CONDA_PY: "27"
+  CMD_IN_ENV: "cmd /E:ON /V:ON /C with_appveyor_visual_studio.cmd"
```

Comment:
Proposing removing this logic. Let's see how that goes and reassess.

xref: https://github.com/conda-forge/conda-smithy/pull/392